### PR TITLE
issue=1167, add r/w sla info in sdk, some corner case statistics

### PR DESCRIPTION
**Pull request by caijieming-baidu**

### CODE REVIEW

**Comment by taocp** [comment]
LGTM

File: src/master/master_impl.cc [hunk 1]
```diff
@@ -4882,7 +4887,10 @@ void MasterImpl::ScanMetaCallbackForSplit(TabletPtr tablet,
     first_meta.set_status(kTableOffLine);
     tablet_manager_->AddTablet(first_meta, TableSchema(), &first_tablet);
 
-    LOG(INFO) << "try load child tablets, \nfirst: " << first_meta.ShortDebugString()
+    tablet_availability_->AddNotReadyTablet(first_tablet->GetPath());
+    tablet_availability_->AddNotReadyTablet(second_tablet->GetPath());
+    LOG(INFO) << "spilit finish, " << tablet << ", try load child tablets, "
```
Comment (00k) [inline src/master/master_impl.cc]: split

Comment (caijieming-baidu) [inline src/master/master_impl.cc]: 已改

**Comment by xupeilin** [comment]
@taocp review

**Comment by taocp** [comment]
@xupeilin  
LGTM

:)

**Comment by 00k** [comment]
修改后增量提commit
评论里带上issue号

**Comment by caijieming-baidu** [comment]
#1167

**Pull request merged**